src/components/admin/services/judging/Toolbar.jsx

```diff
@@ -203,15 +208,12 @@ const Toolbar = ({
         onSubmit={generate}
       >
         <div className="flex flex-row items-center gap-2 pb-3 pl-2">
-          <Input
-            setObject={setInput}
-            object={input}
-            label="rotations"
-            showLabel={false}
-            maxLength={2}
-            placeholder="ie. 5"
-            clear={true}
-          />
+          <InputOTP maxLength={2}>
```

Comment:
needs to handle the logic as well, currently these numbers only get visually seen but not tracked. You can check if the numbers are being tracking by console logging their values in the functions where they are supposed to be called